From 1cb46a0ab2d5f713a0006b1ae7d0bdfc54d5cfe2 Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fabian@ritter-vogt.de>
Date: Fri, 22 May 2020 19:48:34 +0200
Subject: [PATCH] Revert "Make WindowSwitching (Alt+Tab) the default left
 touchscreen edge"

This reverts commit c453eb696cc97208f8ebf2f632995b891bb425c3.

This causes a 1px wide window on the left screen edge
that steals clicks from windows. Very annoying.

BUG: 387775
---
 kcmkwin/kwinscreenedges/kwintouchscreensettings.kcfg | 2 +-
 tabbox/tabbox.cpp                                    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

Index: kwin-5.15.80git.20200807T195756~c8433e519/kcmkwin/kwinscreenedges/kwintouchscreensettings.kcfg
===================================================================
--- kwin-5.15.80git.20200807T195756~c8433e519.orig/kcmkwin/kwinscreenedges/kwintouchscreensettings.kcfg	2020-08-07 21:57:56.000000000 +0200
+++ kwin-5.15.80git.20200807T195756~c8433e519/kcmkwin/kwinscreenedges/kwintouchscreensettings.kcfg	2020-08-07 22:21:56.023358131 +0200
@@ -47,7 +47,7 @@
     </group>
     <group name="TabBox">
         <entry name="TouchBorderActivateTabBox" key="TouchBorderActivate" type="IntList">
-            <default>ElectricLeft</default>
+            <default>ElectricNone</default>
         </entry>
         <entry key="TouchBorderAlternativeActivate" type="IntList">
             <default>ElectricNone</default>
Index: kwin-5.15.80git.20200807T195756~c8433e519/tabbox/tabbox.cpp
===================================================================
--- kwin-5.15.80git.20200807T195756~c8433e519.orig/tabbox/tabbox.cpp	2020-08-07 21:57:56.000000000 +0200
+++ kwin-5.15.80git.20200807T195756~c8433e519/tabbox/tabbox.cpp	2020-08-07 22:21:56.023358131 +0200
@@ -782,7 +782,7 @@
             actions.insert(ElectricBorder(i), a);
         }
     };
-    touchConfig(QStringLiteral("TouchBorderActivate"), m_touchActivate, TabBoxWindowsMode, QStringList{QString::number(int(ElectricLeft))});
+    touchConfig(QStringLiteral("TouchBorderActivate"), m_touchActivate, TabBoxWindowsMode);
     touchConfig(QStringLiteral("TouchBorderAlternativeActivate"), m_touchAlternativeActivate, TabBoxWindowsAlternativeMode);
 }
 
